outlines/serve/serve.py

```diff
@@ -65,10 +66,13 @@ async def generate(request: Request) -> Response:
 
     json_schema = request_dict.pop("schema", None)
     regex_string = request_dict.pop("regex", None)
+    cfg_string = request_dict.pop("cfg", None)
```

Comment:
Maybe name this "grammar" instead of CFG which might not speak to a lot of people.

Reply:
fixed